app.js

```diff
@@ -81,30 +151,58 @@ function clearUI() {
     dinamicContent.innerHTML = '';
 }
 
-function displayDialog(repo) {
+async function displayDialog(repo) {
     dialogContent.innerHTML = `
-        <h2 class="text-center"> ${repo.name} </h2>
-        <div class="dialog__repo-card flex flex-wrap justify-around">
-            <p class="m-0"> 
-            <span class="text-bold"> Owner: </span> ${repo.owner.login} </p>
-            ${repo.language !== null ? (`
+            <h2 class="text-center" > ${repo.name} </h2>
+                <div class="dialog__repo-card flex flex-wrap justify-around">
+                    <p class="m-0">
+                        <span class="text-bold"> Owner: </span> ${repo.owner.login} </p>
+                    ${repo.language !== null ? (`
                 <div class="flex align-center">
                    <p class="m-0"> 
                    <span class="text-bold"> Language</span>: ${repo.language} </p>
                 </div>
             `) : ''}
-        <div class="flex align-center">
-                <a href="https://github.com/${repo.owner.login}/${repo.name}/stargazers" type="button">
-                <span class="text-bold"> Stars: </span>
-                <span> ${repo.stargazers_count} </span>
-                </a>
-        </div>
-        <div class="flex align-center">
-                <a href="https://github.com/${repo.owner.login}/${repo.name}/network/members">
-                <span class="text-bold"> Forks: </span>
-                <span"> ${repo.forks_count} </span>
-                </a>
+                    <div class="flex align-center">
+                        <a href="https://github.com/${repo.owner.login}/${repo.name}/stargazers" type="button">
+                            <span class="text-bold"> Stars: </span>
+                            <span> ${repo.stargazers_count} </span>
+                        </a>
+                    </div>
+                    <div class="flex align-center">
+                        <a href="https://github.com/${repo.owner.login}/${repo.name}/network/members">
+                            <span class="text-bold"> Forks: </span>
+                            <span"> ${repo.forks_count} </span>
+                    </a>
```

Comment:
Same here. It seems this code didn't change, you only changed it's alignment. Let's revert this change.